lib/anycablebility/cli/app.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'logger'
+require 'anycablebility/rpc'
+require 'anycablebility/tests'
+require 'anycablebility/client_factory'
+
+module Anycablebility
+  module Cli
+    # This class is the entry point of Anycablebility
+    class App
+      def run(
+        target: 'ws://0.0.0.0:8080/cable',
+        redis: 'redis://localhost:6379',
+        debug: false
+      )
+        logger = Logger.new(STDOUT)
+        logger.level = debug ? Logger::DEBUG : Logger::WARN
+
+        rpc = Rpc.instance.configure(redis, debug, logger)
```

Comment:
>There's a Logger#formatter=. Maybe we should take advantage of it for that purpose?

Great idea 👍
And we can do that later, let's skip for now.

>attr_accessor :logger on the Anycablebility instead

Yep, you're right